src/web/lib/components/ThemeBuilder/index.js

```diff
@@ -104,28 +104,66 @@ export const ThemeBuilder = props => {
     }
   };
 
+  let tabsElement = null;
+
+  const handleKeyUp = e => {
+    const currentTab = document.activeElement;
+    const currentTabIndex = Array.from(tabsElement.children).indexOf(currentTab);
+
+    let nextTab = null;
+
+    switch (e.key) {
+      case "ArrowLeft": {
+        nextTab = tabsElement.children[Math.max(0, currentTabIndex - 1)];
+        break;
+      }
+      case "ArrowRight": {
+        nextTab = tabsElement.children[Math.min(tabList.length - 1, currentTabIndex + 1)];
+        break;
+      }
+      case "Home": {
+        nextTab = tabsElement.children[0];
+        break;
+      }
+      case "End": {
+        nextTab = tabsElement.children[tabList.length - 1];
+        break;
+      }
+    }
+
+    if (nextTab) {
+      currentTab.setAttribute("tabIndex", -1);
+      nextTab.setAttribute("tabIndex", 0);
+      nextTab.focus();
+    }
+  };
+
   return (
     <div className="theme-builder">
       <div className="theme-builder__tabs-wrapper">
-        <ul className="theme-builder__tabs">
+        <div className="theme-builder__tabs" role="tablist" ref={el => (tabsElement = el)} onKeyUp={handleKeyUp}>
```

Comment:
Nit: There's [a newer ref API in React 16.3](https://reactjs.org/docs/refs-and-the-dom.html#creating-refs) that uses an object created in the constructor rather than a function created at render time. Might be nice to use that here. 

But, of course, we're still using this function form in other places, so 🤷‍♂️

Reply:
The documentation states

> You may not use the ref attribute on functional components because they don’t have instances.

Reply:
Ah, good call, I missed that!